graphql/schema.graphql

```diff
@@ -163,21 +166,31 @@ input UseSpruceOptionsInput {
 
 type Host {
   id: ID!
+  displayName: String!
   hostUrl: String!
   distroId: String
```

Comment:
this field can be removed and be replaced by the `distro` field

Reply:
removed